docs/tokens/stellar-asset-contract.mdx

```diff
@@ -52,12 +52,12 @@ Every Stellar asset on Stellar has reserved a contract address that the Stellar
 
 It can be deployed using the [Stellar CLI] as shown [here](../build/guides/cli/deploy-stellar-asset-contract.mdx).
 
-Or the [Stellar SDK] can be used as shown [here](../learn/encyclopedia/contract-development/contract-interactions/stellar-transaction.mdx#xdr-usage) by calling `InvokeHostFunctionOp` with `HOST_FUNCTION_TYPE_CREATE_CONTRACT` and `CONTRACT_ID_FROM_ASSET`. The resulting token will have a deterministic identifier, which will be the sha256 hash of `HashIDPreimage::ENVELOPE_TYPE_CONTRACT_ID_FROM_ASSET` xdr specified [here][contract_id].
+Or the [Stellar CLI] can be used as shown [here](../learn/encyclopedia/contract-development/contract-interactions/stellar-transaction.mdx#xdr-usage) by calling `InvokeHostFunctionOp` with `HOST_FUNCTION_TYPE_CREATE_CONTRACT` and `CONTRACT_ID_FROM_ASSET`. The resulting token will have a deterministic identifier, which will be the sha256 hash of `HashIDPreimage::ENVELOPE_TYPE_CONTRACT_ID_FROM_ASSET` xdr specified [here][contract_id].
 
 Anyone can deploy the instances of Stellar Asset Contract. Note, that the initialization of the Stellar Asset Contracts happens automatically during the deployment. Asset Issuer will have the administrative permissions after the contract has been deployed.
 
 [contract_id]: https://github.com/stellar/stellar-xdr/blob/dc23adf60e095a6ce626b2b09128e58a5eae0cd0/Stellar-transaction.x#L661
-[stellar-cli]: ../tools/developer-tools/README.mdx#cli
+[stellar cli]: ../tools/developer-tools/README.mdx#cli
```

Comment:
This is a really good catch! Thanks for noticing it.

Looks like the link definition got changed at some point, but the inline text never did. Funny enough, the text _should_ read `Stellar SDK`, rather than CLI. Would you please...

- [ ] Leave the CLI link definition as you have it (`[stellar cli]: link`)
- [ ] Reverse the change on (currently) line 55 so it reads `Or the [Stellar SDK] can ...`
- [ ] Add a new link definition below the CLI one `[stellar sdk]: ../tools/sdks/library`